There is no execinfo.h in musl. Disable backtrace code if
__GLIBC__ is not defined.

--- Source/WTF/wtf/Assertions.cpp
+++ Source/WTF/wtf/Assertions.cpp
@@ -71,8 +71,10 @@
 #if OS(DARWIN) || (OS(LINUX) && !defined(__UCLIBC__))
 #include <cxxabi.h>
 #include <dlfcn.h>
+#if defined(__GLIBC__)
 #include <execinfo.h>
 #endif
+#endif
 
 extern "C" {
 
@@ -225,7 +227,7 @@
 
 void WTFGetBacktrace(void** stack, int* size)
 {
-#if OS(DARWIN) || (OS(LINUX) && !defined(__UCLIBC__))
+#if OS(DARWIN) || (OS(LINUX) && defined(__GLIBC__))
     *size = backtrace(stack, *size);
 #elif OS(WINDOWS)
     // The CaptureStackBackTrace function is available in XP, but it is not defined
index 657ced4..ceb9c47 100644
--- Source/JavaScriptCore/inspector/JSGlobalObjectInspectorController.cpp.orig  2015-09-24 08:04:57.685969077 +0200
+++ Source/JavaScriptCore/inspector/JSGlobalObjectInspectorController.cpp       2015-09-24 08:05:25.307830938 +0200
@@ -46,7 +46,7 @@
 #include <wtf/Stopwatch.h>
 
 #include <cxxabi.h>
-#if OS(DARWIN) || (OS(LINUX) && !PLATFORM(GTK))
+#if OS(DARWIN) || (defined(__GLIBC__) && !PLATFORM(GTK))
 #include <dlfcn.h>
 #include <execinfo.h>
 #endif
